libkbfs/journal_server.go

```diff
@@ -93,26 +105,51 @@ func makeJournalServer(
 	return &jServer
 }
 
+func (j *JournalServer) getDirLocked() string {
+	if j.currentVerifyingKey == (VerifyingKey{}) {
+		panic("currentVerifyingKey is zero")
+	}
+
+	// Device IDs and verifying keys are globally unique, so no
+	// need to have the uid in the path. Furthermore, everything
+	// after the first two bytes (four characters) are randomly
+	// generated, so taking the first 36 characters gives ~1/2^64
+	// collision probability.
```

Comment:
You may have already thought about this already, but I wasted some time thinking about base62 encoding, which would be shorter and have no collisions -- on Linux.  But elsewhere the case insensitivity would hurt us.  base32 is marginally better (~56 characters instead of 70 for the full ID I think), but I'm good with this way.  No action needed, just writing thoughts down.